src/platform/site-wide/user-nav/containers/Main.jsx

```diff
@@ -162,6 +172,10 @@ export class Main extends Component {
     localStorage.setItem(ACCOUNT_TRANSITION_DISMISSED, true);
   };
 
+  closeAccountTransitionSuccessModal = () => {
```

Comment:
If we do decide that the way forward is to store the dismissal of this modal in session storage we would need to add that logic here